sync/config/results.yaml

```diff
@@ -0,0 +1,41 @@
+# Each YAML file under sync/ configures how sync/sync.py synchronizes
+# contents of various versions from its source of truth (usually a GitHub
+# repository of a Tekton component, such as tektoncd/pipelines) to
+# content/ (for the lastest version) and vault/ (for earlier versions).
+
+# The name of the component.
+# sync.py will use this value to build directories in content/ and vault/.
+component: Results
+# The order of the component.
+displayOrder: 3
+# The GitHub repository where documentation resides.
+repository: https://github.com/tektoncd/results
+# The directory in the GitHub repository where contents reside.
+docDirectory: docs
+# The link to the GitHub tag page.
+archive: https://github.com/tektoncd/results/tags
+# The tags (versions) of contents to sync.
+# Note that sync.py and related script reads tags in the order specified in
+# the following list; the first entry in tags will automatically become the
+# latest version of contents.
+# To add a new version, append to the list as below
+# - name: vX.Y.Z
+#   displayName: vX.Y.Z
+#   folders:
+#     docs:
+#       target: '/'            # optional, default value '/'
+#       index: README.md       # optional, if _index.md exists
+#       include: ['*.md']      # optional, default value ['*']
+#       exclude: ['not_in.md'] # optional, default value []
+#       header: <dict>         # optional, no header added if not set
+#         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
+tags:
+- name: main
+  # The name to display on tekton.dev.
+  # sync.py will use this value in the version switcher and other places.
+  displayName: main
+  # Dict of folders to sync
+  folders:
+    docs:
+      index: README.md
```

Comment:
There is no README.md in the docs folder https://github.com/tektoncd/results/tree/main/docs so the sync fails.
You can specify any valid `*.md` file to be the root one. Looking at `install` and `roadmap` it feels like you might need a new file before it's published on the website.

Reply:
good catch! see https://github.com/tektoncd/results/pull/69

Reply:
Thanks!
The sync works now:
![image](https://user-images.githubusercontent.com/2205608/106740050-d5a88380-6611-11eb-9fb8-5eb5c577e39e.png)

However the content is still not linked on the website. I think that's because there are no headers in the original docs.

Reply:
I have a proposed functionality here that takes the `linkTitle` from the filename https://github.com/tektoncd/website/pull/206/files#diff-2c664ae5c58a32f24120a20acf65d8de8fefaf135d3134a7f9641d6d96bc2579R182-R185 - that works for the CLI docs, but it doesn't look of when applied to results, at least the root link should be called "Results" and not "README.md".

![image](https://user-images.githubusercontent.com/2205608/106742016-5f595080-6614-11eb-8cfe-f94034fa0d2a.png)

I could hack something into the sync script to set the link title to the project name for the index page, which would work, but it may be best to just add the headers on the results website docs.

TBH I don't link having those headers in the md files, but the alternative would be to maintain a large map of the required file by file in the sync config, which I'm not sure is very good either.

Reply:
> TBH I don't link having those headers in the md files, but the alternative would be to maintain a large map of the required file by file in the sync config, which I'm not sure is very good either.

I'm not a huge fan of the headers either. Another alternative here (short of @wlynch's original project site proposal) might be to move more of the responsibility for the sync config to the project repos. They could define a config file in the docs root for example that the sync script then consumes instead of having the mapping live in the website repo.